src/krux/pages/home.py

```diff
@@ -54,11 +56,82 @@ def __init__(self, ctx):
 
     def mnemonic(self):
         """Handler for the 'mnemonic' menu item"""
+        submenu = Menu(
+            self.ctx,
+            [
+                (t("Words"), self.display_mnemonic_words),
+                (t("Standard QR"), self.display_standard_qr),
+                (t("Compact SeedQR"), self.display_compact_qr),
+                (t("Transcribe SeedQR"), self.transcribe_compact_qr),
+                (t("Back"), lambda: MENU_EXIT),
+            ],
+        )
+        submenu.run_loop()
+        return MENU_CONTINUE
+
+    def display_mnemonic_words(self):
+        """Displays only the mnemonic words"""
         self.display_mnemonic(self.ctx.wallet.key.mnemonic)
         self.ctx.input.wait_for_button()
+
+    def display_standard_qr(self):
+        """Displays regular words QR code"""
         self.display_qr_codes(self.ctx.wallet.key.mnemonic, FORMAT_NONE, None)
         self.print_qr_prompt(self.ctx.wallet.key.mnemonic, FORMAT_NONE)
-        return MENU_CONTINUE
+
+    def display_compact_qr(self):
+        """Displays binary compact SeedQR code"""
+        code, _ = self._binary_seed_qr()
+        self.ctx.display.draw_qr_code(0, code, bright=True)
+        self.ctx.input.wait_for_button()
+        self.print_qr_prompt(self.ctx.wallet.key.mnemonic, FORMAT_NONE)
+
+    def transcribe_compact_qr(self):
+        """Disables touch and displays compact SeedQR code with grid to help drawing"""
+
+        def draw_grided_qr(grid_size, qr_size):
+            """Draws grided QR"""
+            self.ctx.display.draw_qr_code(0, code, bright=True)
+            grif_offset = self.ctx.display.width() % (qr_size + 2)
+            grif_offset //= 2
+            grid_pad = self.ctx.display.width() // (qr_size + 2)
+            grif_offset += grid_pad
+            for i in range(qr_size + 1):
+                self.ctx.display.fill_rectangle(
+                    grif_offset,
+                    grif_offset + i * grid_pad,
+                    qr_size * grid_pad + 1,
+                    grid_size,
+                    lcd.RED,
+                )
+                self.ctx.display.fill_rectangle(
+                    grif_offset + i * grid_pad,
+                    grif_offset,
+                    grid_size,
+                    qr_size * grid_pad + 1,
+                    lcd.RED,
+                )
+
+        code, qr_size = self._binary_seed_qr()
+        if self.ctx.input.touch is not None:
+            self.ctx.display.draw_hcentered_text(
+                "Touch disabled\nPress Enter twice to leave",
```

Comment:
Could you change this to:
"Touch disabled for transcription.\nPress enter twice to leave."

Also, needs to be translated